src/sentry/explore/models.py

```diff
@@ -19,10 +19,12 @@
 class ExploreSavedQueryDataset(TypesClass):
     SPANS = 0
     OURLOGS = 1
+    DISCOVER_TRANSACTIONS = 101
```

Comment:
nit: should we name this `SEGMENT_SPANS` or something? And add a comment here indicating it's a temporary dataset tracking migrated queries?